lib/helpers.js

```diff
@@ -149,9 +193,20 @@ _beforeEach.givenUserWithRole = function (attrs, role, optionalHandler) {
       }
 
       test.userRole = result;
-      test.app.models.roleMapping.create(
+
+      var roleMappingModelName = test.roleMappingModel ?
+        test.roleMappingModel : DEFAULT_ROLE_MAPPING_MODEL;
+
+      var RoleMapping = test.app.models[roleMappingModelName];
+
+      if (!RoleMapping) {
+        return done('RoleMapping model with name ' +
+          roleMappingModelName + ' not found');
+      }
+
+      RoleMapping.create(
         {principalId: test.user.id,
```

Comment:
I would also suggest to add more tests to test the new and impacted existing methods.